components/builder-originsrv/src/migrations/origins.rs

```diff
@@ -117,5 +117,26 @@ pub fn migrate(migrator: &mut Migrator) -> Result<()> {
                         UPDATE origins SET session_sync = true WHERE id = in_origin_id;
                     END
                     $$ LANGUAGE plpgsql VOLATILE"#)?;
+    migrator.migrate("originsrv",
```

Comment:
pretty sure the key needs to be incremented to something like `originsrv-v7`

Reply:
Nah, that hasn't been required since the migrator was rewritten months ago.

Reply:
oh ok!